parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
+        - "--oidc-client-id=<oidcClientId>"
+        - "--oidc-issuer-url=<oidcIssuerUrl>"
+        - "--oidc-username-claim=oid"
```

Comment:
@jackfrancis Makes sense, fixed. Those lines with 'oidc' will get removed if 'aadProfile' is not present.